### PR TITLE
App dispatcher

### PR DESCRIPTION
**Pull request by perlindgren**
just a test

### CODE REVIEW

File: src/parse/app.rs [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
     spanned::Spanned,
-    ExprParen, Fields, ForeignItem, Ident, Item, Lit, Path, Token, Visibility,
+    Expr,
+    ExprArray,
+    ExprParen,
+    Fields,
+    ForeignItem,
+    Ident,
+    Item,
+    //Lit,
```
Comment (AfoHT) [inline src/parse/app.rs]: When we're satisfied with the functionality the unused imports could be removed

Comment (perlindgren) [reply src/parse/app.rs]: Fixed along with other stuff.

**Review by AfoHT — APPROVED** [review]
bors merge

**Comment by AfoHT** [comment]
bors retry

**Pull request merged**